include/qpid/dispatch/atomic.h

```diff
@@ -203,6 +223,14 @@ static inline void sys_atomic_destroy(sys_atomic_t *ref)
     sys_mutex_free(ref->lock);
 }
 
+static inline void sys_atomic_release_fence()
+{
+    //it shoudn't leak any OS resources, because (unshared) spin locks
+    //are just user space atomic ints
+    sys_spinlock_t lock;
+    sys_spin_init(&lock);
```

Comment:
This can be seen here: https://github.com/lattera/glibc/blob/895ef79e04a953cac1493863bcae29ad85657ee1/sysdeps/x86_64/nptl/pthread_spin_unlock.S#L22